src/makeRobotAccountant.js

```diff
@@ -7,6 +7,19 @@
 
 function makeRobotAccountant() {
   // write code here
+  let count = 0;
+
+  return (x) => {
+    count++;
+
+    return (y) => {
+      if (count < 3 || count % 2) {
+        return x + y;
+      } else {
+        return 'Bzzz... Error!';
+      }
```

Comment:
ternary operator here  will make a little bit less code, just comment. I'm not sure that we shoud use it here but I did